.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 build
 coverage.xml
 .phpunit.result.cache
+.idea/
```

Comment:
I'd like to keep editor specific items out of here. You can also add this to your global .gitignore file - See https://sebastiandedeyne.com/setting-up-a-global-gitignore-file/

Reply:
Removed `.idea` from `.gitignore`.